src/process.py

```diff
@@ -27,6 +28,7 @@ def __init__(self, cmd, shell=False, env=None, capture=True):
         kw = dict(shell=shell, close_fds=True)
         if not shell:
             cmd = shlex.split(cmd)
+            cmd.append("--data=" + f'{settings.prepare_temp_settings(settings.sett())}')
```

Comment:
I am glad that it works, but it looks for me that it should be at different place.

We choose `cmd` for slicing somewhere [here](https://github.com/epit3d/spycer/blob/ad4deea72901f6c31ca076278c9f92a819b2f8f6/src/controller.py#L506) and such preparation to pass `--data` should be there, not in more or less general-purpose class